src/kOS.Safe/Compilation/Opcode.cs

```diff
@@ -744,7 +744,7 @@ public class OpcodeSetMember : Opcode
         public override void Execute(ICpu cpu)
         {
             Structure value = cpu.PopStructureEncapsulated();         // new value to set it to
-            string suffixName = cpu.PopStack().ToString().ToUpper();  // name of suffix being set
+            string suffixName = cpu.PopStack().ToString();            // name of suffix being set
```

Comment:
The case sensitivity change is being done for the sake of memory allocation more than it is for actual execution time.  I actually doubt that the time is faster, at best it could probably be equal time if the case matches.  The big reason for doing it is to cut down on the number of local scope orphan strings we allocate.  Every time that you call `ToUpper` another string is allocated, and since we discard it right away it gets tossed to the GC.  kOS by it's nature is already fairly garbage heavy anything that we don't store in a variable is essentially thrown to the GC.  So I'm just trying to minimize the number of objects we throw away, thereby reducing how frequently the GC needs to run.

In the grand scheme of things, it probably isn't a huge change either way.  And our string comparisons by far are not the biggest bottle neck,

You'd be surprised though.  Because of how often we throw away objects, we are constantly recreating suffix lists, and the list results of those suffixes.  That's why I implemented caching of the `ship` variable elsewhere, and caching the value of `parts`.  It's also why we defer making the suffix list on strings, scalars, and booleans, since there is a good chance you won't even try to access a suffix of those values instead performing math on them.

TLDR: I don't think it is faster, but it's more memory efficient.